code/modules/client/preferences.dm

```diff
@@ -88,7 +88,11 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 	/// If set to TRUE, will update character_profiles on the next ui_data tick.
 	var/tainted_character_profiles = FALSE
 
+	/// the timerid of the char_preview/updatebody() callback
+	var/character_preview_view_timerid
```

Comment:
this name is bad and you're using the comment to do some of the heavy lifting
rather then naming the id after the object it is targeting, it seems more sensible to name it after the thing the timer does